lib/defaults.js

```
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  CANT_FIND: 'I can\'t find the address in the list',
+  CHANGE: 'Change',
+  POSTCODE_ERROR: {
+    'not-found': 'Sorry – we couldn’t find any addresses for that postcode, enter your address manually',
+    'cant-connect': 'Sorry – we couldn’t connect to the postcode lookup service at this time, enter your address manually'
+  }
+}
```

could these be translations?

---

they are, these are fallbacks

---

if lookup fails (you forget or CBA to include a translation value) these are used as defaults. I think a standalone component/behaviour should have fallbacks for things like these to minimise setup in the majority of use cases

---

I half agree and half disagree.

By failing early and showing the untranslated keys then there's an obvious cue to the user that they need to configure it properly, where these fallbacks are not really "proper".

However, I think it would be good to have some way of automating importing "default" translations for modules in the same way as we do with `hof-template-partials`. That would need some thought to not become unwieldy though.